frontend/src/components/ElectionForm/ElectionDetails.tsx

```diff
@@ -140,74 +151,111 @@ export default function ElectionDetails({ election, applyElectionUpdate, getStyl
-                            if (e.target.value == null || e.target.value == '') {
-                                applyElectionUpdate(election => election.start_time = undefined)
-                            } else {
-                            applyElectionUpdate(election => 
-                                election.start_time = DateTime.fromISO(e.target.value).setZone(timeZone,{keepLocalTime : true}).toJSDate())
-                            }
-
-                        }}
-                    />
-                    <FormHelperText error sx={{ pl: 0, mt: 0 }}>
-                        {errors.startTime}
-                    </FormHelperText>
-                </FormControl>
-            </Grid>
-            <Grid item xs={6} sx={{ m: 0, p: 1 }} justifyContent='center'>
-                <FormControl fullWidth>
-                    <InputLabel shrink>
-                        Stop Date
-                    </InputLabel>
-                    <Input
-                        type='datetime-local'
-                        error={errors.endTime !== ''}
-                        value={dateToLocalLuxonDate(election.end_time, timeZone)}
-                        onChange={(e) => {
-                            setErrors({ ...errors, endTime: '' })
-                            if (e.target.value == null || e.target.value == '') {
-                                applyElectionUpdate(election => { election.end_time = undefined })
-                            } else {
-                                applyElectionUpdate(election =>  
-                                election.end_time = DateTime.fromISO(e.target.value).setZone(timeZone,{keepLocalTime : true}).toJSDate())
-                            }
-                        }}
-                    />
-                    <FormHelperText error sx={{ pl: 0, mt: 0 }}>
-                        {errors.endTime}
-                    </FormHelperText>
-                </FormControl>
+
+            <Grid item xs={12} sx={{ m: 0, p: 1 }}>
+
+                <FormGroup>
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                checked={enableStartEndTime}
+                                onChange={(e) => {
+                                    setEnableStartEndTime(e.target.checked)
+                                    if (e.target.checked) {
+                                        applyElectionUpdate(election => {
+                                            election.start_time = DateTime.now().setZone(timeZone, { keepLocalTime: true }).toJSDate()
+                                            election.end_time = DateTime.now().plus({ days: 1 }).setZone(timeZone, { keepLocalTime: true }).toJSDate()
+                                        })
+                                    }
+                                    else {
+                                        applyElectionUpdate(election => {
```

Comment:
nit

Looks like unchecking and checking the box clears the start and end time? I feel like it would be a bad experience if someone accidentally unchecked the box and then lost their dates. Ideally it should remember the previously set date when someone rechecks the box

Fixing this would add some complexity since we'd need a to add a boolean to the election settings for whether start/end times are enabled, so I think this is fine for now, but I wanted to bring it up anyway

Reply:
I think if we wanted that I would just put the values in a state hook, then when enabled it would use that state as a default. I'll think on this a bit.